cdhweb/people/tests/test_models.py

```diff
@@ -85,6 +86,280 @@ def test_str(self):
         assert str(self.person) == "tom jones"
 
 
+class TestPersonQuerySet(TestCase):
+
+    def setUp(self):
+        """create testing data"""
+        # FIXME why is this necessary?
+        Title.objects.all().delete()
```

Comment:
curious if you ran into this also, it seemed liked some titles were already in the database when tests ran for some reason

Reply:
I don't remember this, must be from your code that I cherry-picked over. Do we preload titles anywhere in data migrations? maybe for exec?

Reply:
yep, that was it! we create them in `cdhweb/people/migrations/0006_exec_committee_role.py`. good catch. I guess we should just switch the tests/fixtures to `get_or_create()`.